src/components/about-us/section-01/title.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import styled from 'styled-components'
+import { screen } from '../utils/screen'
+
+
+const TitleWrapper = styled.div`
+  display: flex;
```

Comment:
could this be done by `display: inline-block` ?

Reply:
Yes we can. I'll use `display: inline-block` instead.